Userland/Libraries/LibTLS/TLSv12.cpp

```diff
@@ -115,6 +115,19 @@ bool Certificate::is_valid() const
     return true;
 }
 
+// https://www.ietf.org/rfc/rfc5280.html#page-12
+bool Certificate::is_self_signed() const
```

Comment:
(this could also be stored as a variable – as the work for given certificate needs to be done once?)

Reply:
I though about it, but was unable to find a way that worked (I'm not a C developer).

Reply:
(still: might stay as is)